lunar/distribution.yaml

```diff
@@ -675,5 +675,21 @@ repositories:
       url: https://github.com/ros/std_msgs.git
       version: groovy-devel
     status: maintained
+  urdfdom_py:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.2
```

Comment:
There is a prompt for the doc entry when you run bloom. When it's generating the pull-request. Note that it will seed it with the previous value. Usually it's a development branch so it only needs updating when the development forks.

Note that the upstream uri is malformed. it should end in .git not a slash: https://github.com/ros/urdf_parser_py/  https://github.com/ros-gbp/urdfdom_py-release/commit/b4a45a156e933005685df783c8f6cb2bddb3bf5e#diff-b9cba95a33d64aec1406327c4d43fa8dR115 though it looks like that should be fixed for all the versions. Though there's enough manually done in this package that it appears not to be a problem.